__tests__/app.test.js

```diff
@@ -39,4 +39,35 @@ describe("GET /api/categories", () => {
   });
 });
 
-
+describe.only("GET /api/reviews", () => {
```

Comment:
much like Haz mentioned not adding console.logs to main I would also recommend not having .onlys in your tests when pushing, just so you know that all of your tests are passing 👍